website/snippets/_cloud-environments-info.md

```diff
@@ -118,3 +112,14 @@ Partial parsing in dbt Cloud requires dbt version 1.4 or newer. The feature does
 To enable, select **Account settings** from the gear menu and enable the **Partial parsing** option.
 
 <Lightbox src="/img/docs/deploy/example-account-settings.png" width="85%" title="Example of the Partial parsing option" />
+
+### Account access to Advanced CI features <Lifecycle status="beta" />
+
+To help increase the governance and improve the quality of the data, you can set up automation that tests code changes before merging them into production with [CI jobs](/docs/deploy/ci-jobs). You can also enable Advanced CI features, such as [compare changes](/docs/deploy/continuous-integration#compare-changes), that allow dbt Cloud account members to view details about the changes between what's currently in your production environment and the pull request's latest commit, providing observability into how data changes are affected by code changes.
+
+To use Advanced CI features, your dbt Cloud account must have access to them. Ask your dbt Cloud administrator to enable Advanced CI features on your account, which they can do by selecting **Account settings** from the gear menu and choosing the **Enable account access to Advanced CI** option.
```

Comment:
```suggestion
Enable Advanced CI features on your account by selecting **Account settings** from the gear menu and choosing the **Enable account access to Advanced CI** option.
```

Reply:
for this particular option, an admin has to give access so want to make that clear